src/api/spec/controllers/source/key_info_controller_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Source::KeyInfoController, type: :controller do
+  describe 'GET #show' do
+    let(:user) { create(:confirmed_user) }
+    let(:project) { create(:project, name: "test_project", title: "Test Project") }
+    let(:backend_url) { CONFIG['source_url'] + Project::KeyInfo.send(:backend_url, project.name) }
+    let(:gpg_public_key) { Faker::Lorem.characters(1024) }
+    let(:ssl_certificate) { Faker::Lorem.characters(1024) }
```

Comment:
Sure, however, I expected to be used in the actual test. But I'm fine with leaving it as it is if you prefer...